docs/advanced/storagenetwork/storagenetwork.md

```diff
@@ -0,0 +1,240 @@
+---
+sidebar_position: 1
+sidebar_label: Storage Network
+title: ""
+---
+
+# Storage Network
+
+Harvester uses Longhorn as its built-in storage system to provide block device volumes for VMs and Pods. If the user wishes to isolate Longhorn replication traffic from the Kubernetes cluster network (i.e. the management network) or other cluster-wide workloads. Users can allocate a dedicated storage network for Longhorn replication traffic to get better network bandwidth and performance.
+
+For more informations, please refer [Longhorn Storage Network](https://longhorn.io/docs/1.3.2/advanced-resources/deploy/storage-network/)
+
+:::note
+
+Configuring Longhorn settings directly is not recommended, as this can lead to untested situations.
+
+:::
+
+## Prerequisites
+
+There are some prerequisites before configuring the Harvester Storage Network setting.
+
+- Well-configured Cluster Network and VLAN Config
+    - Users have to ensure the Cluster Network is configured and VLAN Config will cover all nodes and ensure the network connectivity is working and expected in all nodes. 
+- All VMs should be stopped.
+    - We recommend checking the VM status with the following command and should get an empty result.
+    - `kubectl get -A vmi`
+- All pods that are attached to Longhorn Volumes should be stopped.
+    - Users could skip this step with the Harvester Storage Network setting. Harvester will stop Longhorn-related pods automatically.
+
+## Configuration Example
+
+- VLAN ID
+	- Please check with your network switch setting, and provide a dedicated VLAN ID for Storage Network
+- Well-configured Cluster Network and VLAN Config
+	- Please refer Networking page for more details and configure `Cluster Network` and `VLAN Config` but not `Networks`.
+- IP range for Storage Network
+	- IP range should not conflict or overlap with Kubernetes cluster networks(both `10.42.0.0/16` and `10.43.0.0/16` are reserved).
```

Comment:
Also `10.53.0.0/16` and `10.52.0.0/16`. Our RKE2 configuration:
```
cluster-cidr: 10.52.0.0/16
service-cidr: 10.53.0.0/16
```

Reply:
fix